Caltech-IPAC/firefly
# IRSA-964: TimeSeries tool: downloading images from selected rows is broken

- Converted code relying on an ipac table file as source of the table data.
- Added unit testing for EmbeddedDbUtil
https://caltech-ipac.atlassian.net/browse/IRSA-964

Additional changes are in ife and firefly_test_data under the same branch.

To test:
http://localhost:8080/firefly/ts.html
- upload a file to view its time series data.
- select multiple rows then select "Prepare Download"
- the images from the selected rows should be downloaded.
This was not working prior to this PR.
